rust/processor/src/db/parquet/models/token_v2_models/v1_token_royalty.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::token_v2_models::raw_v1_token_royalty::{
+        CurrentTokenRoyaltyV1Convertible, RawCurrentTokenRoyaltyV1,
+    },
+};
+use allocative_derive::Allocative;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct CurrentTokenRoyaltyV1 {
+    pub token_data_id: String,
+    pub payee_address: String,
+    pub royalty_points_numerator: String, // String format of BigDecimal
+    pub royalty_points_denominator: String, // String format of BigDecimal
+    pub last_transaction_version: i64,
+    #[allocative(skip)]
+    pub last_transaction_timestamp: chrono::NaiveDateTime,
+}
+
+impl NamedTable for CurrentTokenRoyaltyV1 {
+    const TABLE_NAME: &'static str = "current_token_royalty_v1";
```

Comment:
I wish this was `royalties` but I imagine there are existing dependencies using this name already

Reply:
I think this can be renamed to `royalties` since there is no existing parquet processor processing this table. will rename this to royalties

Reply:
awesome. lgtm